Cp 50 Image Upload and Client Preview Page

aatbip opened the pull request:
### Task/Ticket

[Image Upload and Client Preview Page](https://outsidetech.atlassian.net/browse/CP-50?atlOrigin=eyJpIjoiZmFhMjdjYTYxNTZkNGU3YTlhNDg5MGFkYjU2OTE1MTIiLCJwIjoiaiJ9)

#### The main changes are

- [x] Added banner image upload
- [x] Added editor image upload
- [x] Added client preview page

saugatmaharjan commented on src/app/components/ClientPreview.tsx:
```diff
@@ -0,0 +1,111 @@
+  })
+
+  useEffect(() => {
+    if (!editor) return
```
no null or undefined return

saugatmaharjan commented on src/app/components/EditorInterface.tsx:
```diff
@@ -204,8 +207,26 @@ const EditorInterface = () => {
   }, [appState?.appState.settings])
 
+  useEffect(() => {
+    ;(async () => {
```
`;`

aatbip replied:
seems like prettier automatically puts this

saugatmaharjan commented on src/app/components/EditorInterface.tsx:
```diff
@@ -204,8 +207,26 @@ const EditorInterface = () => {
+        setBannerImage(appState?.appState.bannerImgUrl as string)
+      }
+    })()
+  }, [appState?.appState.bannerImgUrl])
```
this variable is in two useEffect dependency array, can we put the login in single useEffect @aatbip ?

saugatmaharjan commented on src/components/autofillFields/AutofillFields.tsx:
```diff
@@ -4,7 +4,7 @@ import { staticAutofillValues } from '@/utils/constants'
 import { Editor } from '@tiptap/react'
 import { When } from '../hoc/When'
-import { useEffect, useState } from 'react'
+import { useEffect } from 'react'
```
import order

aatbip commented:
https://github.com/pagevamp/copilot-client-home/pull/15/commits/fcabc6fa9d625cc51527a088d59db51e4941465e commit solves the addressed reviews @saugatmaharjan
